docs/config.md

```diff
@@ -149,6 +149,12 @@ The explanation and default value of each configuration item are as follows:
     # conf_dir is the directory in which the admin places a CNI conf.
     conf_dir = "/etc/cni/net.d"
 
+    # max_conf_num is the max number of CNI plugin config file will load
+    # By default, the containerd will load only 1 CRI plugin. If you want load multiple CRI plugins,
+    # set max_conf_num > 1. You should know exactly each CRI plugin do, whether they are compatible.
+    # max i<= 0 means no limit.
```

Comment:
typo nits:
    # max_conf_num specifies the maximum number of CNI plugin config files that will be loaded from the CNI config directory.    
    # By default, containerd will load only 1 CNI plugin config file. If you want to load multiple CNI plugin config files set max_conf_num to the number desired or 0, as 0 means no limit (load all CNI plugin config files from the CNI config directory).